sources/p987.6/document.adoc

```diff
@@ -0,0 +1,36 @@
+= Draft Recommended Practice for Preparing an IEEE Standards Draft
+:docnumber: P987.6/D3
+:doctype: standard
+:copyright-year: 2020
+:revdate: 2021-01-01
+:confirmed-date: 2021-01-01
```

Comment:
Why `confirmed-date`? Nothing is confirmed in a draft standard.

Reply:
Huh, I thought it was the way to enter an approved date:

![capture1](https://user-images.githubusercontent.com/33627611/177040678-18b0f37d-3edb-4e9c-b153-271441b34508.PNG)

I took it from the generic attributes batch:

![capture2](https://user-images.githubusercontent.com/33627611/177040700-f7cdc0f4-03bf-40ef-b327-f187cfde42f3.PNG)

Reply:
Attribute removed.

Reply:
I see, thanks for the clarification!